src/frontend/staff-portal/src/app/components/paging/paging.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="main-pagination">
+    <div class="pagination">
+        <a [class.disabled]="currentPage === 1" (click)="changePage(currentPage - 1)"><<</a>
+        <ng-container *ngFor="let e of [].constructor(totalPages); let i = index">
```

Comment:
Can the `[].constructor(totalPages)` move to `.ts` ?

Reply:
Done